src/app_model/types/_keys/_keybindings.py

```diff
@@ -19,6 +19,58 @@
 _re_cmd = re.compile(r"cmd[\+|\-]")
 
 
+def get_keys_user_facing_representation(
```

Comment:
probably don't need a function to re-define these dicts every time a string is generated, since they won't change at all at runtime, the only thing is which dict to pull from... so maybe a module level:

```python
WIN_KEY_SYMBOLS: dict[str, str] = {...}
MACOS_KEY_SYMBOLS: dict[str, str] = {**WIN_SYMBOLS, "Ctrl": "⌃", "Alt": "⌥", "Meta": "⌘"}
LINUX_KEY_SYMBOLS: dict[str, str] = {**WIN_SYMBOLS, "Meta": "Super"}

# same for names
```

and then wherever you would have previously used this function, just pick a dict

Reply:
(however, see later comment)